v2/policyfile.go

```diff
@@ -95,7 +95,7 @@ type ACLDERPNode struct {
 	STUNPort         int    `json:"stunPort,omitempty" hujson:"STUNPort,omitempty"`
 	STUNOnly         bool   `json:"stunOnly,omitempty" hujson:"STUNOnly,omitempty"`
 	DERPPort         int    `json:"derpPort,omitempty" hujson:"DERPPort,omitempty"`
-	InsecureForTests bool   `json:"insecureForRests,omitempty" hujson:"InsecureForTests,omitempty"`
+	InsecureForTests bool   `json:"InsecureForTests,omitempty" hujson:"InsecureForTests,omitempty"`
```

Comment:
Why do we even have this field at all? It's only meant to be used for tests, hence the "ForTests".

How about we just delete this line entirely?

/cc @oxtoacart @mpminardi

Reply:
I tried starting with a lowercase letter but couldn't save it. I suspect it's a bug in the official website API.